CLAUDE.md

```
@@ -57,6 +57,21 @@ bash scripts/smoke-readme.sh ghcr.io/askdba/myvector:mysql8.4
 ./scripts/test-online-updates.sh ghcr.io/askdba/myvector:mysql8.4
 ```
 
+**Pre-release gate (run before every tag):**
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)